src/config/renderer.js

```diff
@@ -1,5 +1,8 @@
 // Static setter for the renderer
-export function setRenderer(renderer) {
+export function setRenderer(renderer, options) {
   this.prototype._renderHtml = renderer;
+  if (options && options.destructive === false) {
```

Comment:
I think the feature could be good, though I don't like the name.

Maybe just call this `reInitializeRegion` or maybe `preserveRegion` so the flag check can be `options.preserveRegion`.

Reply:
I'm fine changing the name but should not be region specific. This flag may be needed for other things than reinitialize region like for example adding an onReady event which would have a different behavior from destructive and non destructive regions.

Reply:
What about `preventDestroy`? It's used in stuff like `region.show` and it feels like it does a similar job.